tests/test_dsp_filtering.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+import sparrowpy as sp
+import pyfar as pf
+
```

Comment:
```suggestion

@pytest.mark.parametrize("frequencies",[
    np.array([1000]),
    np.array([1000, 2000]),
    np.array([500,1200,6700,15200]),
])
@pytest.mark.parametrize("frac",[
    1,3,
])
@pytest.mark.parametrize("rms",[
    1, (1,1), (2,1,2),
])

def test_band_filtering_multi_dim(frequencies,frac,rms):
    """Test freq band data estimation."""

    noise = pf.signals.noise(n_samples=441, rms=rms)

    band_sig,_ = sp.dsp.band_filter_signal(
        signal=noise, frequencies=frequencies, num_fractions=frac)

    assert band_sig.cshape==(noise.cshape + (len(frequencies),) )

```
mutidim input is not test, like this eg.